hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/BackendStore.java

```diff
@@ -57,7 +57,22 @@
     public void rollbackTx();
 
     // Get metadata by key
-    public <R> R metadata(HugeType type, String meta, Object[] args);
+    public default <R> R metadata(HugeType type, String meta, Object[] args) {
+        BackendSession session = this.session(type);
+        if (type == null) {
+            return this.metaDispatcher().dispatchHandler(session, meta, args);
+        } else {
+            BackendTable table = this.table(type);
+            return table.metaDispatcher().dispatchHandler(session, meta, args);
+        }
```

Comment:
prefer to add a local var dispatcher
return dispatcher.dispatchHandler(session, meta, args);